examples/GUI/GuiTools.js

```diff
@@ -17,12 +17,23 @@ dat.GUI.prototype.removeFolder = function removeFolder(name) {
     this.onResize();
 };
 
-function GuiTools(domId) {
+function GuiTools(domId, view) {
     this.gui = new dat.GUI({ autoPlace: false });
     this.gui.domElement.id = domId;
     viewerDiv.appendChild(this.gui.domElement);
     this.colorGui = this.gui.addFolder('Color Layers');
     this.elevationGui = this.gui.addFolder('Elevation Layers');
+
+    if (view) {
+        view.addEventListener('layers-order-changed', () => {
+            for (const layer of view.getLayers(l => l.type === 'color')) {
+                this.removeLayersGUI(layer.id);
+            }
+
+            const colorLayers = view.getLayers(l => l.type === 'color');
+            this.addImageryLayersGUI(colorLayers);
```

Comment:
Not sure what you mean by `it needs to move globe`.*
I've only implemented auto-updating for the `layers-order-changed` event because that's the only one working in all View (other event are `GlobeView` specific currently).